index.html

```diff
@@ -49,5 +52,170 @@ <h3>I am a Software Developer</h3>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+	</section>
+
+	<section class="about-me">
+		<div class="container2">
+			<h1>About Me</h1>
+			<p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and 
+			   experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			   <button class="btn">Get My Resume</button>
+			   <img src="images/about img.png" alt="Vector shapes">
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/diamond.png" alt="">Language</h3>
+			<ul>
+				<li>Javascript</li>
+				<li>Ruby</li>
+				<li>Html</li>
+				<li>Css</li>
+			</ul>
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/Rectangle.png" alt="">Frameworks</h3>
+			<ul>
+				<li>Bootstrap</li>
+				<li>Ruby on rail</li>
+				<li>RSPec</li>
+				<li>Copy Bara</li>
+				<li>Selenium</li>
+			</ul>
+		</div>
+
+		<div class="tools">
+			<h3><img src="images/circle.png" alt="">Skills</h3>
+			<ul>
+				<li>Codekit</li>
+				<li>Github</li>
+				<li>Codepen</li>
+				<li>Gitlab</li>
+				<li>Terminal</li>
+			</ul>
+		</div>
```

Comment:
- [ ] Please follow the Figma design your design is not matching to Figma design check below the difference :point_down: 

You Design:
![screencapture-127-0-0-1-5500-2023-02-19-03_17_28 (1)](https://user-images.githubusercontent.com/45226848/219901939-2779d1cc-a9f9-4eee-ae58-ea682bd0d955.png)

Figma Design:
![Screenshot from 2023-02-19 03-24-55](https://user-images.githubusercontent.com/45226848/219902038-227196f7-ddfb-462c-9ed7-76ea94af017b.png)